src/rp2040_flash.rs

```diff
@@ -42,3 +45,43 @@ pub fn read_device_config_from_rp2040_flash() -> &'static [u8] {
     let addr = (FLASH_XIP_BASE + FLASH_END - FLASH_USER_SIZE) as *const u8;
     unsafe { slice::from_raw_parts(addr, FLASH_USER_SIZE as usize) }
 }
+
+pub fn read_is_audio_from_rp2040_flash() -> bool {
+    let addr = (FLASH_XIP_BASE + FLASH_END - FLASH_USER_SIZE) as *const u8;
+    let config = unsafe { slice::from_raw_parts(addr, 5 as usize) };
+    if config[0] == u8::MAX && config[1] == u8::MAX && config[2] == u8::MAX && config[3] == u8::MAX
+    {
+        false
+    } else {
+        config[4] == 1
+    }
+}
+
+pub fn read_alarm_from_rp2040_flash() -> &'static [u8] {
```

Comment:
So is this writing the alarm data to the very last page of the flash?  Would be good to be explicit about that in a comment.  Also seems like if our user flash usage of `FLASH_USER_SIZE` increased (device config got bigger for example?), we could risk colliding with this?

Reply:
added a comment, and yes we would risk colliding with this, as it stands if our config was larger than a sector 4096 bytes it would break anyway. Could write the alarm to the first sector and config to the second sector to avoid possible problems

Reply:
I think a comment is fine for now :)